packages/core/template_project/app/build.gradle

```
@@ -158,6 +163,12 @@ android {
     }
 }
 
+<% if (serviceAccountJson) { %>
+play {
+    serviceAccountCredentials.set(file("<%= serviceAccountJson %>"))
```

I think this should read `serviceAccountJsonFile` now?
```suggestion
    serviceAccountCredentials.set(file("<%= serviceAccountJsonFile %>"))
```

---

Yes it should

---

There were a few other places that it looks like I missed with this change.